js/image.js

```diff
@@ -0,0 +1,19 @@
+(function ($) {
+	var retries = 5;
+	$( document).ready(function(){
+		$('img').bind('error', function() {
+			var image = $(this);
+			var imgSrc = image.attr('src');
+			if (retries > 0 && typeof imgSrc !== undefined) {
+				window.setTimeout(function(){
+					image.attr('src', imgSrc);
+					retries--;
+				},1000);
+			} else {
+				image.unbind('error');
+				image.attr('title', 'The image could not be retrieved, please contact Bynder Support');
+				image.attr('alt', 'The image could not be retrieved, please contact Bynder Support');
+			}
+		});
+	});
+})(jQuery);
```

Comment:
I see tabs instead of spaces, some spaces missing after brackets etc. But also, it is not really clear to me what this code should do so please add a small explanation in comments.